sql/2022/fonts/font_palettes_colors.sql

```diff
@@ -0,0 +1,35 @@
+SELECT
+  client,
+  entries,
+  pages,
+  total,
+  pages / total AS pct
+FROM (
+  SELECT
+    client,
+    JSON_EXTRACT_SCALAR(payload,
+      '$._font_details.color.numPaletteEntries') AS entries,
```

Comment:
Same question. Use percentiles for high cardinality data.

Reply:
This one also has low cardinality. I think I'll just mention that most fonts only have a single palette and most palettes have only a handful of colors. I simplified the queries a bit, but I don't think percentiles would be super helpful until there is a lot more cardinality.